### ISSUE TITLE
Separate context Menu

### ISSUE THREAD

**Issue by Shinkir0** [gyng/save-in#77]
Make as in this add-on https://github.com/ssborbis/ContextSearch-web-ext
if you long press the right mouse button (or other hotkey), a separate menu with folders for saving appears.

Is this possible?

**Comment by gyng** [comment]
It's possible, but will take a good chunk of work. Probably not in the near future.

Is this for convenience? If so, there is a workaround to move the menu up to the top in https://github.com/gyng/save-in/issues/21#issuecomment-354538673. There's also movement in the bug for accesskeys in Firefox. If accesskeys still don't show up after a while I'll consider adding something like this.

**Comment by Shinkir0** [comment]
Yes, I think it will be much more convenient. Than less need to do movements to the ultimate goal the better.

I used to use DragIT to save pictures. There you had to pull the picture and there was a menu with pre-configured folders, it was also very convenient, but DragIT died a long time ago.

_(googletranslate)_

**Comment by tombones** [comment]
I'm afraid the menu relocation workaround linked there isn't working for me. I'm definitely using the right profile for userChrome.css because another fix (enabling scrollwheel control of the tab bar) does work.

![image](https://user-images.githubusercontent.com/3936058/43688552-ee4836ca-98b8-11e8-998c-ff2da2b11cd1.png)

![image](https://user-images.githubusercontent.com/3936058/43688545-ccf8f7ac-98b8-11e8-8b56-1977b977796c.png)

**Comment by gyng** [comment]
You could try https://github.com/gyng/save-in/issues/21#issuecomment-349250207 as well. I can't guarantee if it'll work.
